test/features/navigation.feature

```diff
@@ -1,17 +1,5 @@
 Feature: Navigation
 
-    Scenario: Check for the presence of the 'Request data' link in header when visiting as a non-logged in user
```

Comment:
We should also split tests with labels to use the matrix for publications and data

Reply:
Eh, these tests are largely a sanity-check.